lib/plugin-manager.js

```diff
@@ -53,9 +53,26 @@ function getPluginsForGithubClasses(plugins, classList) {
 
 export default class {
   constructor(plugins) {
+    this._pluginsList = plugins;
     this._plugins = buildPluginCache(plugins);
   }
 
+  getClickHandler(pluginName) {
+    const plugin = this._pluginsList.find(pluginClass => pluginClass.name === pluginName);
+
+    if (!plugin) {
+      console.error(`Plugin ${pluginName} not found`);  // eslint-disable-line no-console
+      return;
+    }
+
+    if (!plugin.clickHandler) {
+      console.error(`No clickHandler found for ${pluginName}`);  // eslint-disable-line no-console
+      return;
+    }
+
+    return plugin.clickHandler;
```

Comment:
In the spirit of https://github.com/OctoLinker/browser-extension/pull/274/files#r99658228, I think `clickHandler` should instead be `resolve` on this line and in the previous conditional.